index.js

```diff
@@ -132,20 +134,20 @@ Plex.prototype.debugLog = function(string)
 Plex.prototype.httpHandler = function(self, body) {
     var jsonStart = body.indexOf("{");
     var json = body.substring(jsonStart, body.indexOf("\n", jsonStart));
-	var event;
-	try {
-    	event = JSON.parse(json);
-	}
-	catch(e) {
-		self.debugLog("Webhook URL called without JSON body.");
-	}
-	
-	if (!event)
-	{
-		return;
-	}
-	
-	self.debugLog("Plex incoming webhook");
+    var event;
+    try {
+        event = JSON.parse(json);
+    }
+    catch(e) {
+        self.debugLog("Webhook URL called without JSON body.");
+    }
+    
+    if (!event)
+    {
+        return;
+    }
+    
+    self.debugLog("Plex incoming webhook");
```

Comment:
Thanks for the indentation fix. 💜

Reply:
Truthfully, a plugin did it without me realising :-)